client/components/title-format-editor/index.jsx

```diff
@@ -0,0 +1,307 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import {
+	map,
+	max,
+	min
+} from 'lodash';
+
+// The following polyfills exist for the draft-js editor, since
+// we are unable to change its codebase and yet we are waiting
+// on a solid solution for general IE polyfills
+//
+// They were borrowed and modified from MDN
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith
+if ( ! String.prototype.endsWith ) {
```

Comment:
I think this is a good solution for now, but it should definitely be short lived. We should follow through with more comprehensive polyfill strategies after this is merged.